app/serializers/user_serializer.rb

```diff
@@ -274,6 +276,14 @@ def stats
     UserAction.stats(object.id, scope)
   end
 
+  def ignored
+    IgnoredUser.find_by(user_id: scope.user.try(:id), ignored_user_id: object.id).present?
```

Comment:
Does this generate an N+1?

Reply:
For a single User loaded, we will try to find ONE `user_ignored` db record that belongs to the current logged in User.

I think potentially N+1 if we are loading a list of Users and using this Serializer, but I see that we are using this Serializer for loading a single user, like in the `show` method of the UserController.